Makefile

```diff
@@ -239,6 +240,15 @@ flexdll_initer_gnat.o: flexdll_initer.c
 flexdll_initer_mingw64.o: flexdll_initer.c
 	$(MIN64CC) $(GCC_FLAGS) -c -o $@ $<
 
+build_config.ml: Makefile
```

Comment:
FWIW, just update version.ml with the extra definitions (as things stand, you've not `build_config.ml` to the `clean` target but as `version.ml` already had the mingw ones there 🤷‍♀️)